source/MapPanel.cpp

```diff
@@ -118,8 +118,8 @@ namespace {
 			// Get the system in which the planet storage is located.
 			const Planet *planet = hold.first;
 			const System *system = planet->GetSystem();
-			// Skip outfits stored on planets without a system.
-			if(!system)
+			// Skip outfits stored on planets without a system or an outfitter.
+			if(!system || !planet->HasOutfitter())
```

Comment:
if your changes are right, there should be no outfits stored on planets without outfiter